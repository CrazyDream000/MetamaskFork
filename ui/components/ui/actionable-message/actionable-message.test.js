import React from 'react';

import { renderWithProvider } from '../../../../test/jest';
import ActionableMessage from '.';

const createProps = (customProps = {}) => {
  return {
    message: 'I am an actionable message!',
    ...customProps,
  };
};

describe('ActionableMessage', () => {
  it('renders the component with initial props', () => {
    const props = createProps();
    const { container, getByText } = renderWithProvider(
      <ActionableMessage {...props} />,
    );
    expect(getByText(props.message)).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
