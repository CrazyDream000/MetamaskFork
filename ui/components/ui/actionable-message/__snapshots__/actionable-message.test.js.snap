// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActionableMessage renders the component with initial props 1`] = `
<div>
  <div
    class="actionable-message"
  >
    
    <div
      class="actionable-message__message"
    >
      I am an actionable message!
    </div>
  </div>
</div>
`;
